decentraland/unity-explorer
# feat: real authentication UI flow integrated with dApp

### What?

Implements the real user web3 authentication through auth-dapp: https://github.com/decentraland/auth-server?tab=readme-ov-file
Integrates [Socket.IO](https://github.com/itisnajim/SocketIOUnity) for the server communication and transport.

### How to test?

1. Run `DynamicSceneLoading` scene
2. Click on Web3 Authentication Login button from the debug panel
3. A new browser tab should be opened after a short delay, requesting a wallet signature
4. Sign the authentication with your wallet
5. Go back to the client and check that the world initializes as expected
